scheduler.module

```diff
@@ -344,17 +344,17 @@ function _scheduler_translation_validate($form, FormStateInterface $form_state)
     $publishing_enabled_types = array_keys($plugin->getEnabledTypes('publish'));
     $unpublishing_enabled_types = array_keys($plugin->getEnabledTypes('unpublish'));
     $enabled = [];
-    foreach ($content_types as $name => $settings) {
```

Comment:
This change was because the function already had a variable `$settings` used earlier. It's confusing to redefine a variable if not necessary.